src/citrine/resources/api_error.py

```diff
@@ -4,7 +4,7 @@
 from gemd.entity.dict_serializable import DictSerializable
 
 
-class ValidationError(DictSerializable):
+class ValidationError(DictSerializable, typ="validation_error"):
```

Comment:
DictSerializable is being modified to make objects w/o typ abstract according to ABC.  There is no type field in the object that comes in.  Might be reasonable to just migrate this to a Serializable[] object.

Reply:
It's a smell to have this citrine-python Error object inheriting its type from GEMD

Reply:
Probably better to rebuild from scratch.  This seemed like the highest-velocity solution.

Reply:
Classes rebuilt as per _serialization.